packages/editor/src/editor/sidebar/fields/TextareaField.tsx

```diff
@@ -12,14 +14,40 @@ type TextareaFieldProps = {
 
 export const TextareaField = ({ label, value, onChange, message }: TextareaFieldProps) => {
   const [open, setOpen] = useState(false);
+  const [savedSelection, setSavedSelection] = useState<{ start: number; end: number } | undefined>();
+  const textAreaRef = useRef<HTMLTextAreaElement>(null);
+
   return (
     <Dialog open={open} onOpenChange={setOpen}>
       <BasicField label={label} message={message}>
         <Flex direction='row'>
-          <Textarea value={value} onChange={e => onChange(e.target.value)} autoResize={true} />
-          <DialogTrigger asChild style={{ marginLeft: '-25px', marginTop: '8px' }}>
-            <Button icon={IvyIcons.ListSearch} aria-label='Browser' />
-          </DialogTrigger>
+          <Textarea
+            value={value}
+            onChange={e => onChange(e.target.value)}
+            autoResize={true}
+            ref={textAreaRef}
+            onSelect={() => handleTextSelection(textAreaRef, setSavedSelection)}
+          />
+          <Flex
+            direction='row'
+            style={{
+              marginLeft: canPopoverBeRendered(textAreaRef, savedSelection) && savedSelection ? '-49px' : '-25px',
+              marginTop: '8px'
+            }}
```

Comment:
what is that?

Reply:
For the moment I have to solve it this way, because you can't place a button in the area of a textarea. I would have to extend the Textarea component, or make a kind of TextareaGroup, because if I were to use the InputGroup here, it would look like this:
![grafik](https://github.com/user-attachments/assets/b7a375fe-d933-469b-93b0-cf97cc113faa)

And if I would not add an InputGroup or this marginLeft/bottom, it would look like this:
![grafik](https://github.com/user-attachments/assets/3a7d980a-4281-4c77-afd2-3a5266b05367)

Reply:
ahh right. thanks for clarification